package-readmes/animationHelpers.md

```diff
@@ -2,425 +2,416 @@
 
+```js
+import { zoom } from '@velo/wix-animation-helpers';
+```
 
-![alt_text](https://d2x3xhvgiqkx42.cloudfront.net/12345678-1234-1234-1234-1234567890ab/b4ae9703-4364-4550-a797-1cc95ed8046e/2021/02/10/6f816214-66b0-403d-a239-98e11f295a18/182c62a1-0016-46c5-a4fd-ed9d51d25175.png 'image_repeater_element_1_image_and_button')
+3. Run your code in the `$w.onReady()` function.
+4. To zoom in to multiple elements of the same type, use a [type selector](https://www.wix.com/velo/reference/$w/$w) and call the `zoom()` function using this selector. We will call ours ‘VectorImage’. The zoom function will zoom in to all elements with the ‘VectorImage’ type selector. We will also set the animation to zoom in to the image elements from a scale of 1 to 2, and for a duration of 2 seconds.  
 
 
-![alt_text](https://d2x3xhvgiqkx42.cloudfront.net/12345678-1234-1234-1234-1234567890ab/b4ae9703-4364-4550-a797-1cc95ed8046e/2021/02/10/fcc4da39-d42a-4abc-acb2-8dfdb7880898/d709405c-5e42-4714-b986-30cf4d4fc84c.png 'image_repeater_element_2_image_only')
+    ```js
+    zoom('VectorImage', { duration: 2000, from: 1, to: 2 });
+    ```
 
 
-![alt_text](https://d2x3xhvgiqkx42.cloudfront.net/12345678-1234-1234-1234-1234567890ab/b4ae9703-4364-4550-a797-1cc95ed8046e/2021/02/10/9c2585c7-2b6a-4a31-b37c-4b24ee0adc53/776825b7-7a0d-4f4c-914a-089190e9915e.png 'image_repeater_element_2_button_only')
+##### Zoom an element on hover 
 
 
 
-1. Add the following page elements to your site:
-    +  3 repeaters. Each repeater item contains an image and a button. 
+1. Add a text element to your site. 
+2. Import the `zoom` function from the package to your page. 
+
+    ```js
+    import { zoom } from '@velo/wix-animation-helpers';
+    ```
 
-2. Import the `initRepeaterAnimation` and `runRepeaterAnimation` functions from the package to your page.
 
+3. Run your code in the `$w.onReady()` function.
+4. To zoom in to the text element on hover, we first need to stop the animation function from playing by default. To do this, call the `zoom()` function with the text element and change the default value of the `pause` parameter from `false` to `true`. We will also change the text element to zoom in for a duration of .5 seconds.  
+
+    ```js
+    const text = $w('#zoomText');
+    const textZoomTimeLine = zoom(text, { duration: 500, pause: true });
     ```
-    import { initRepeaterAnimation, runRepeaterAnimation } from '@velo/wix-animation-helpers';
+
+5. Add the `onMouseIn()` event handler to the text element. This function will run when you hover over the text element. Call the `textZoomTimeLine.play()` function in the event handler to play the animation on hover .   
+Add the `onMouseOut()` event handler to the text element. Call the `textZoomTimeLine.reverse()` function in the event handler to reverse the animation when you are no longer hovering over the text. 
+
+    ```js
+    text.onMouseIn(() => textZoomTimeLine.play());
```

Comment:
This block of code isn't showing properly. Play around with js tags and/or see where I placed them in the github package readme repo.